pkg/sdk/api/v1alpha1/thanos_types.go

```diff
@@ -239,18 +239,61 @@ type StoreGateway struct {
 	GRPCGracePeriod string `json:"grpcGracePeriod,omitempty" thanos:"--grpc-grace-period=%s"`
 	// Maximum size of items held in the in-memory index cache.
 	IndexCacheSize string `json:"indexCacheSize,omitempty" thanos:"--index-cache-size=%s"`
+	// Path to YAML file that contains index cache configuration. See format details:
+	// https://thanos.io/tip/components/store.md/#index-cache
+	IndexCacheConfigFile string `json:"indexCacheConfigFile" thanos:"index-cache.config-file=%s"`
+	// Alternative to 'index-cache.config-file' flag (lower priority). Content of YAML file that contains index cache configuration. See format details:
+	// https://thanos.io/tip/components/store.md/#index-cache
+	IndexCacheConfig string `json:"indexCacheConfig" thanos"--index-cache.config=%s"`
```

Comment:
Are these really required or you just left out the omitempty?

Reply:
Nice catch thank you!